### ISSUE TITLE
Show thumbnails when adding root and favorites

### ISSUE THREAD

**Issue by melvin4u445** [ARK-Builders/ARK-Navigator#215]
Is there a reason why we are not showing thumbnails when adding root and favorites?

![image](https://user-images.githubusercontent.com/17034768/153993141-5b84f35d-9aa5-447a-b01f-e1473f8bf561.png)

**Comment by kirillt** [comment]
Yes, we disabled it for now. But should be implemented eventually.

Technical difficulty here is that we store previews/thumbnails by id, not by path. And before we add folder we can't have ids because we haven't started indexing yet. But we can invent something for sure.
